src/cpuid/src/template/intel/t2.rs

```diff
@@ -33,26 +36,31 @@ fn update_feature_info_entry(entry: &mut kvm_cpuid_entry2, _vm_spec: &VmSpec) ->
         .write_bit(ecx::MONITOR_BITINDEX, false)
         .write_bit(ecx::DS_CPL_SHIFT, false)
         .write_bit(ecx::VMX_BITINDEX, false)
+        .write_bit(ecx::SMX_BITINDEX, false)
```

Comment:
Is it worth calling out the changes to the `t2` template in the changelog?

Reply:
Yes, I'll add a callout to the Changelog.